src/main/java/Duke.java

```diff
@@ -1,10 +1,113 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner scanner = new Scanner(System.in);
+
+        String logo =
+                "\t __   __\n"
+                        + "\t|  | |  |   ____     _  _   _    _\n"
+                        + "\t|  |_|  |  / _  \\   | |/_\\ | |  | |\n"
+                        + "\t|   _   | | |_|  \\  |  /   | \\_/  |\n"
+                        + "\t|__| |__|  \\___/\\_\\ |_|     \\__/|_|\n";
+
+        String divider = "\t____________________________________________________________";
+        String spacer = "\t";
+        System.out.println(divider);
+        System.out.println(logo);
+        System.out.println(spacer+"Hello! I'm Haru");
+        System.out.println(spacer+"What can I do for you?");
+        System.out.println(divider);
+
+        String userCommand = scanner.nextLine();
+
+        boolean isExit = false;
+        Task[] storedUserTasks = new Task[100];
+        int taskIndex;
+        String description, by, from, to;
+        int userTextCount = 0;
+        while(!isExit){
+            switch (userCommand.split(" ")[0]){
+            case "list":
+                System.out.println(divider);
+                System.out.println(spacer+"Here are the tasks in your lists:");
+                for(int i=0;i<userTextCount;i++){
+                    System.out.println(spacer+(i+1)+"."+storedUserTasks[i].toString());
+                }
+                System.out.println(divider);
+                userCommand = scanner.nextLine();
+                break;
+            case "bye":
+                isExit = true;
+                break;
+            case "mark":
+                taskIndex = Integer.parseInt(userCommand.split(" ")[1]);
```

Comment:
Maybe you can use methods to handle different situations, so that the abstraction of the code will be better..